website/docs/r/cloud_project_database_m3db_user.html.markdown

```diff
@@ -61,13 +61,9 @@ The following arguments are supported:
 
 * `service_name` - (Required, Forces new resource) The id of the public cloud project. If omitted,
   the `OVH_CLOUD_PROJECT_SERVICE` environment variable is used.
-
 * `cluster_id` - (Required, Forces new resource) Cluster ID.
-
 * `group` - (Optional) Group of the user:
-
-* `name` - (Required, Forces new resource) Name of the user. A user named "avnadmin" is map with already created admin user instead of create a new user.
-
+* `name` - (Required, Forces new resource) Name of the user. A user named "avnadmin" is map with already created admin user instead of creating a new user.
```

Comment:
```suggestion
* `name` - (Required, Forces new resource) Name of the user. A user named "avnadmin" is mapped with already created admin user instead of creating a new user.
```